makefiles/simulators/Makefile.nvc

```diff
@@ -8,6 +8,7 @@ clean::
 
 else
 
+#TRACE :=--vhpi-trace
```

Comment:
Yes I think,
It was just a quick and dirty modification to test nvc on my AXI design. But finally ghdl worked better for my design.
I will improve it this week to use an env var instead of Makefile modification.